src/Runner.Worker/ActionCommandManager.cs

```diff
@@ -183,6 +183,41 @@ public sealed class SetEnvCommandExtension : RunnerService, IActionCommandExtens
 
         public void ProcessCommand(IExecutionContext context, string line, ActionCommand command, ContainerInfo container)
         {
+
+            var serverUrl = context.GetGitHubContext("server_url");
+            var isGHES = false;
```

Comment:
Now that I think of it, I'd rename this variable to `isNotGithub`, or something like that. Our internal environments are not github.com, but also not GHES, so currently what the current is saying is not what the code is actually doing for those environments.